meta/main.yml

```diff
@@ -0,0 +1,5 @@
+# NOTE: dummy file to make ansible-galaxy put the whole content under ROLES_PATH/$name,
```

Comment:
How is this file used? What ansible-galaxy command can or cannot make use of this file?

Reply:
The existence of this file allows to download the role(s) via `ansible-galaxy`, it just needs a [small workaround](https://github.com/zinfra/cailleach/blob/master/environments/exxon-dev/ansible/sft/ansible.cfg#L8). A shart explanation can be found in the last bullet of [this](https://github.com/zinfra/cailleach/commit/5701e7412d2fc07fcd3b5b1a65b0dace09aacf2c) commit message.

Reply:
It would be great if "how to download this with ansible-galaxy (with a workaround)" was documented either in this file, or, even better, in the README of this github repository.

Knowing that this file exists for a workaround without knowing the workaround isn't very useful. Please link to (or write) publicly accessible documentation for this.

Reply:
You are right. I tried to explain [this](https://github.com/wireapp/ansible-sft/pull/8/commits/22ad187aeffe49ec803e3016a28da7083747b9eb).